frontend/app/src/store/assets/retrieval.ts

```diff
@@ -142,13 +142,24 @@ export const useAssetInfoRetrieval = defineStore(
       }
     };
 
+    const getIdentifierBySymbol = (
```

Comment:
I guess the comments I made in the usages of this function kind of apply here too.